lib/build.js

```diff
@@ -266,10 +285,29 @@ export default {
 
       let stdout = '';
       let stderr = '';
+      let liveErroMatchQueue = '';
+      let liveErroMatchHandle = target.liveErrorMatch ? function (d) {
```

Comment:
So why do end matching at all? If we do live error matching we might as well skip the final step (since the last data buffer dropping in will cause the error matching to run).